changelog.txt

```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.67 - released xxx 2022
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* The backend code for MIDI playback has been changed and real pause logic has been
+  re-enabled. Please be aware that things such as application volume and audio quality
+  may change depending on your configuration from ≤0.66
+* The original demo gameplay video now plays when the CorsixTH main menu is idle (this
+  can be disabled in the configuration file)
+* File browser menus now have an OK button (at last!)
+* Selecting an existing file in the Save Game/Map browsers will now update the text
+  input box to that file name
+
+# Changes
+* Handymen now always stay in their assigned plots, except when going to rest
+* Staff members now have a maximum salary, which will be configurable in custom levels
```

Comment:
will be configurable or can be configured? would rather not comment on future features here.